playbooks/roles/jumpbox/files/mongo_db_restore.sh

```diff
@@ -0,0 +1,72 @@
+#fyi: this is an unmodified fork from rex pilot. #todo: fix this
+
+#!/bin/bash
```

Comment:
Add note that this isn't functional. Disable execution on this (ie: put an early exit)